bitnami/contour/crds/ingressroutes.yaml

```diff
@@ -1,373 +0,0 @@
-apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
I'm wondering what would happen with resources already created based on this CRD.

Reply:
Helm does not delete CRDs
> There is no support at this time for upgrading or deleting CRDs using Helm. This was an explicit decision after much community discussion due to the danger for unintentional data loss. Furthermore, there is currently no community consensus around how to handle CRDs and their lifecycle. As this evolves, Helm will add support for those use cases.
Ref: https://helm.sh/docs/chart_best_practices/custom_resource_definitions/

Reply:
When helm would delete CRDs, this change would be dangerous

Reply:
you are right!